### ISSUE TITLE
PRB0040990 CITZ - SILVER - Hardware incident involving MCS-SILVER-APP-14.DMZ

### ISSUE THREAD

**Issue by wmhutchison** [BCDevOps/developer-experience#4987]
**Describe the issue**
This ticket will track the effort spent towards troubleshooting and repairing hardware issues with MCS-SLVER-APP-14.DMZ

**Additional context**
Vendor case number is 5383672770.
Ticket for DXC DCM to open hardware removal ticket: RITM0188341
RFC for performing replacement this Friday: CHG0061640

**How does this benefit the users of our platform?**
Timely resolution of hardware issues brings maximum capacity availability to the users of the SILVER platform.

**Definition of done**
- [x] Vendor case opened and forensics uploaded.
- [x] Wait for vendor support to confirm parts replacement.
- [x] Create supporting tickets for the work (RFC, Data Centre ticket for hardware removal and escorted access if required).
- [x] Execute RFC.
- [x] Bring server back online.

**Comment by wmhutchison** [comment]
All paperwork now created. Just waiting on the following:
- RFC needs Product Owner approval, will poke Luke if not approved by lunch-time.
- Vendor support needs to give us the engineer's name so DCM can update their Equinix ticket to include Escorted access if required.
- Vendor support needs to confirm that the requested date/time is viable so if needed we can modify the RFC and Equinix ticket accordingly.

**Comment by wmhutchison** [comment]
Will poke vendor support at 2pm today if no response is received prior to that time. The DCM ticket as submitted cannot be fulfilled until we know the vendor on-site engineer's name first, and also need to know if the RFC as created/approved for Friday is going to be viable or not.

**Comment by wmhutchison** [comment]
Current Nagios and AlertManager silences for this node are all set to expire this Friday at 2pm. Feel free to extend if desired, or do so once the alerts start coming again. Barring any last-minute vendor updates to the contrary, it's not looking good for getting hardware replaced on Friday, unless vendor support follows up with a confirmation that Friday is still possible, but later in the day. RFC change window is good for the day.

**Comment by StevenBarre** [comment]
HPE was unable to get the parts in time for the CHG and we'll be doing it tomorrow (Saturday).

RITM0188481 Data Centre Request: Vendor Hardware Removal
opened for the tech to remove the defective parts.

**Comment by StevenBarre** [comment]
HPE replaced the power supply and the server booted. They decided we don't need to replace the main board. They will keep the main board with them for the weekend in case the server dies again and they can come back and replace it.

When booting the server, it seemed to be hanging on a CRI-O Auto Update service https://github.com/cri-o/cri-o/blob/main/contrib/systemd/crio-wipe.service so I booted into Rescue Mode https://access.redhat.com/solutions/5500131 and manually cleaned out the `/var/lib/containers/` folder. https://access.redhat.com/solutions/5350721 This took a while, as there was a lot of cached images and pods in there. After that completed, rebooted the server back into normal running mode and it booted quickly.

Cleaned up the orphaned pods left over from the hard power off.

Leaving the server cordoned until after we've made some good progress on the Trident upgrade next week.
